server/http/passport/facebook.coffee

```diff
@@ -64,7 +73,7 @@ app.get '/auth/facebook', passport.authenticate 'facebook',
 
 app.get '/auth/facebook/callback', passport.authenticate 'facebook',
   display: 'touch'
-  successRedirect: '/'
-  failureRedirect: '/login'
+  successRedirect: '/setup'
```

Comment:
can we make sure there is actually a boilerplate setup view in the product-boiler so that this works?